analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Monitor.TryCatch.vb

```diff
@@ -0,0 +1,151 @@
+﻿Imports System.Threading
+
+Namespace Monitor_TryCatch
+
+    Class Program
+
+        Private Condition As Boolean
+        Private Obj As New Object()
+
+        Public Sub Method1(Arg As String)
+            Monitor.Enter(Obj) ' FN
+            Try
+                Console.WriteLine(Arg.Length)
+            Catch ex As Exception
+                Monitor.Exit(Obj)
+                Throw
+            End Try
+        End Sub
+
+        Public Sub Method2(Arg As String)
+            Monitor.Enter(Obj) ' FN
+            Try
+                Console.WriteLine(Arg.Length)
+            Catch ex As Exception
+                Monitor.Exit(Obj)
+                Throw
+            End Try
+        End Sub
```

Comment:
Isn't this exactly the same as `Method1`?

Reply:
I've removed it from both